src/test/news.test.js

```diff
@@ -0,0 +1,219 @@
+const request   = require('supertest');
+const app       = require('../app');
+const jwt       = require('jsonwebtoken');
+const path      = require('path');
+const { Entry } = require('../models');
+require('dotenv').config();
+
+const login = () => {
+  const payload = {
+    dataValues: {
+      roleId: 1,
+      email: 'test@test.com'
+    }
+  }
+
+  const token = jwt.sign(payload, process.env.SECRET_JWT_SEED);
+  return token
+}
+
+const createEntry = async(name) => {
+  await Entry.destroy({where:{}});
+  const token    = login();
+  const filePath = path.join(__dirname, './test-data/image.jpg');
+  const response = await request(app)
+      .post('/news')
+      .set('Authorization', 'bearer ' + token)
+      .field('name', name)
+      .field('content', 'This is a test content')
+      .attach('image', filePath)
+
+  return response
+}
+
+
+describe('POST /news testing', () => {
+
+  it('can only be accessed if the user is signed in as an admin', async() => {
+    await request(app)
+    .post('/news')
+    .send({})
+    .expect(401)
+  })
+  
+  it('returns an error if an invalid name or content is provided', async() => {
+    const token = login();
+  
+    const resp  = await request(app)
+        .post('/news')
+        .set('Authorization', 'bearer ' + token)
+        .send({
+            name:'',
+            content: ''
+        })
+        .expect(400)
+  
+    expect(resp.body.errors[0].msg).toBe('name field is required');
+    expect(resp.body.errors[1].msg).toBe('content field is required');
+  })
+
+  it('returns an error if a file is attached and its not an image', async() => {
+    const token    = login();
+    const filePath = path.join(__dirname, './test-data/word.docx');
+
+    const resp = await request(app)
+        .post('/news')
+        .type('form')
+        .set('Authorization', 'bearer ' + token)
+        .field('name', 'This is a test name')
+        .field('content', 'This is a test content')
+        .attach('image', filePath)
+        .expect(400)
+
+    expect( resp.body.error[0].msg ).toBe('El archivo debe ser una imagen.')
+
+  })
+
+  it('creates an entry with valid inputs', async() => {
+    const response = await createEntry('This is a test name');
+    expect(response.status).toBe(201);
+  })
+
+})
+
+describe('GET /news testing', () => {
+
+  it('returns an object with an entries property', async() => {
+    const resp = await request(app)
+    .get('/news')
+    .expect(200)
+  
+    expect(resp.body.entries).not.toBeUndefined();
+  })
+
+  it('returns an entry idetified by id', async() => {
```

Comment:
Tenes armado el caso de prueba si probas con un ID que no existe?

Reply:
<img width="902" alt="Captura de Pantalla 2022-07-30 a la(s) 3 43 05 p m" src="https://user-images.githubusercontent.com/74872179/181995629-df6ab60a-2e64-49f0-a98a-0caf862be09a.png">

- GET test added if the id does not exist

Reply:
Excelente la evidencia actualizada :)